app/controllers/research_sessions/questions_controller.rb

```diff
@@ -42,6 +42,15 @@ def question_params
       params.require(:research_session).permit(ResearchSession::Steps::PARAMS[step]).tap do |p|
         p['methodologies']&.reject!(&:blank?)
         p['recording_methods']&.reject!(&:blank?)
+
+        %w[
+          travel_expenses_limit
+          food_expenses_limit
+          other_expenses_limit
+          incentive_value
+        ].each do |attr|
```

Comment:
I wonder if we ought to factor this into `ResearchSession.currency_params` or if that's a bit premature